blog/12-12-2022-advance-hardhat-configration-on-celo-using-plugins/advance-hardhat-configration-on-celo-using-plugins.md

```diff
@@ -0,0 +1,751 @@
+---
+title: Advance hardhat configration on celo using plugins
```

Comment:
Or make sure your local forked repo is up-to-date, use git pull upstream main ,with that you will have your repo synced